### DISCUSSION TITLE
Documentation doesn't contain (or I can't see) what response variables definitions are?

### DISCUSSION THREAD

**Discussion by AllTerrainAction** [amberelectric/public-api#4] [General]
For example https://api.amber.com.au/v1/sites/siteid/prices/current?resolution=30

The returned results, I assume channelType = feed in solar vs general use which is clearly understood but what about things like spotPerKwh, spikeStatus, estimate ? An idea what information this gives us will also help in knowing what we can do with that data in our applications. 

Either I missed it or didnt, see below for example.

![image](https://user-images.githubusercontent.com/36219843/122700536-d1ae9280-d28e-11eb-8248-f8615fa3d3f5.png)

**Comment by madpilot** [comment]
Hello! It's in the documentation. At the bottom, there are definitions for all the Objects:

![image](https://user-images.githubusercontent.com/19585/122700987-a8423680-d28f-11eb-9b1f-87f09bc0b921.png)

**Reply by AllTerrainAction** [reply]
Yep, blind lol. As like before thanks Myles.

We've implemented this into our current application and its live - see how it goes now. Thank you for prompt response.

Release for BETA came quicker than you said, so thank you.
